src/crimson/os/seastore/omap_manager/btree/omap_btree_node_impl.cc

```diff
@@ -193,6 +193,70 @@ OMapInnerNode::rm_key(omap_context_t oc, const std::string &key)
   });
 }
 
+OMapInnerNode::iterate_ret
+OMapInnerNode::iterate(
+  omap_context_t oc,
+  ObjectStore::omap_iter_seek_t &start_from,
+  std::function<ObjectStore::omap_iter_ret_t(std::string_view, std::string_view)> &f)
+{
+  LOG_PREFIX(OMapInnerNode::iterate);
+  if (start_from.seek_position != "") {
+    DEBUGT("seek_positon: {}, this: {}", oc.t,
+            start_from.seek_position, *this);
+  } else {
+    DEBUGT("this: {}", oc.t, *this);
+  }
+  auto start_iter = start_from.seek_position != "" ?
+    get_containing_child(start_from.seek_position):
+    iter_cbegin();
+  auto end_iter = iter_cend();
+  return seastar::do_with(
+    start_iter,
+    end_iter,
+    iter_t(start_iter),
+    ObjectStore::omap_iter_ret_t{ObjectStore::omap_iter_ret_t::NEXT},
+    [this, &start_from, oc, &f](
+      auto &start_iter,
+      auto &end_iter,
+      auto &iter,
+      auto &ret)
+    {
+      return trans_intr::repeat(
+        [&, oc, this]() -> iterate_iertr::future<seastar::stop_iteration>
```

Comment:
As discussed before, default lambda capsures such as `=` `&` should be expanded to the actual variables, otherwise it's error-prone in the future maintainences.

Same to the other places.